PaddleCV/PaddleDetection/docs/DATA.md

```diff
@@ -41,8 +41,8 @@ Loads `COCO` type datasets with directory structures like this:
   |   ...
   ```
 
-- Pascal VOC data source
-Loads `Pascal VOC` like datasets with directory structure like this:
+- PASCAL VOC data source
```

Comment:
Both are okay.  According to my observations, "Pascal VOC" is used more often actually

Reply:
I change to Pascal VOC